modules/gdscript/gdscript_tokenizer.cpp

```diff
@@ -164,6 +164,7 @@ bool GDScriptTokenizer::Token::is_identifier() const {
 	switch (type) {
 		case IDENTIFIER:
 		case MATCH: // Used in String.match().
+		case CONST_INF: // Used in Vector2.INF and Vector3.INF
```

Comment:
The comment should also include `Vector4.INF`

Reply:
Fixed it! Thanks!